pkg/reconciler/instances/ory/hydra/hydra_test.go

```diff
@@ -0,0 +1,187 @@
+package hydra
+
+import (
+	"context"
+	k8smocks "github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes/mocks"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+	appv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/utils/pointer"
+	"testing"
+	"time"
+)
+
+const (
+	testNamespase = "kyma-system"
+)
+
+func Test_TriggerSynchronization(t *testing.T) {
+	logger := zaptest.NewLogger(t).Sugar()
+	t.Parallel()
+	t.Run("Should trigger synchronization if hydra-maester is behind hydra", func(t *testing.T) {
+		// given
+		hydraStartTimePod1 := time.Date(2021, 10, 10, 10, 10, 10, 10, time.UTC)
+		hydraStartTimePod2 := time.Date(2021, 10, 10, 10, 10, 7, 10, time.UTC)
+		hydraStartTimePod3 := time.Date(1900, 10, 10, 10, 10, 7, 10, time.UTC)
+		hydraMasesterPodStartTime := time.Date(2021, 10, 10, 10, 10, 6, 10, time.UTC)
+		kubeclient := fakeClient()
+		addPod(kubeclient, "hydra1", "hydra", hydraStartTimePod1, t, v1.PodRunning)
+		addPod(kubeclient, "hydra2", "hydra", hydraStartTimePod2, t, v1.PodRunning)
+		addPod(kubeclient, "hydra3", "hydra", hydraStartTimePod3, t, v1.PodFailed)
+		createDeployment(kubeclient, "ory-hydra-maester", hydraMasesterPodStartTime, t)
+		addPod(kubeclient, "hydra-maester1", "hydra-maester", hydraMasesterPodStartTime, t, v1.PodRunning)
+		client, _ := kubeclient.Clientset()
+
+		// when
+		err := NewDefaultHydraSyncer().TriggerSynchronization(context.TODO(), client, logger, "kyma-system")
```

Comment:
Why don't you use `testNamespace` here? It could be used instead of typing it raw

Reply:
human factor ;-)